.github/workflows/cd.yml

```diff
@@ -15,16 +17,19 @@ jobs:
       with:
         node-version: 12
         registry-url: https://registry.npmjs.org/
+
+    # first push the version to git to avoid publishing smth that is not pushed yet 
+    # (which can happen if there is another commit while this workflow is still running)
+    - run: npm version prerelease -m "[ci skip] %s"
+    - run: git config --global user.name "GitHub CD bot"
```

Comment:
I think we need to do this config before hitting `version prerelease` with `-m` because that will cause git to make a commit (and presumably it's not yet configured?)

Reply:
nice catch, thanks!